### PR TITLE
Fix ErrorElement statement since ErrorElement as moved from AdminBundle to CoreBundle

### PR DESCRIPTION
**Pull request by huitiemesens**

### CODE REVIEW

File: Admin/Extension/Gedmo/TranslatableAdminExtension.php [hunk 1]
```diff
@@ -30,7 +30,7 @@ protected function getTranslatableListener(AdminInterface $admin)
     {
         if ($this->translatableListener == null) {
             $this->translatableListener = $this->getContainer($admin)->get(
-                'stof_doctrine_extensions.listener.translatable'
+                'gedmo.listener.translatable'
```
Comment (rande) [inline Admin/Extension/Gedmo/TranslatableAdminExtension.php]: This should be part of another PR as it is not related to the classname issue.

Comment (huitiemesens) [inline Admin/Extension/Gedmo/TranslatableAdminExtension.php]: That s a mistake i ll dix This and PR again

**Comment by huitiemesens** [comment]
i commit a fix , now it s fine .

**Comment by rande** [comment]
@huitiemesens can you squash you commit ? Also we need to wait for @nicolas-bastien, to merge the PR as this introduce a BC break.

**Comment by huitiemesens** [comment]
okay i squashed commits and repushed ( sorry for these mistakes , usually i don t manage git repos but i wanted to help ;)
